Add FreeBSD install instructions (#698)
tenzir/vast: INSTALLATION.md

### FreeBSD

Installation on FreeBSD currently requries a source build because a VAST port
does not exist (yet).

#### Service Management

FreeBSD uses the [rc(8)][rc] system of startup scripts during system
initialization and for managing services. VAST ships with an rc script (similar
to the ones in `/etc/rc.d` and `/usr/local/etc/rc.d`) that allows
[service(8)][service] to manage a `vast` process with the `start`, `stop`, and
`restart` options. The rc script uses [daemon(8)][daemon] to manage the `vast`
process.

The following steps install VAST as a persistent service:

1. Copy the rc script to `/usr/local/etc/rc.d`.

>    **Note**: this happens automatically when executing the `install` target.

Reviewer: ```suggestion
   **Note**: this happens automatically when running `make install` or `ninja install`.
```
`target` is build system jargon that users may not be familiar with.

Author: I was trying to avoid saying `make` and `ninja`. Maybe we should have all examples be with `make` and mention once that they can be replaced with `ninja`?

Reviewer: That would be acceptable.

> 2. Add the following line to `/etc/rc.conf`:

Reviewer: ```suggestion
2. Add the following line to `/etc/rc.conf` to start VAST on each boot:
```

Author: I don't think this is necessary to add here, because the instructions are for adding VAST as a "persistent service".

Author: I don't think this is necessary to add here, because the instructions are for adding VAST as a "persistent service".

Reviewer: This is fine then.
